resources/views/platform/components/game/game-list.blade.php

```diff
@@ -1,22 +1,34 @@
 @props([
-    'consoles' => [],
-    'games' => [],
-    'sortOrder' => 'title',
+    'availableCheckboxFilters' => [],
+    'availableSelectFilters' => [],
     'availableSorts' => [],
-    'filterOptions' => [],
-    'availableFilters' => [],
     'columns' => [],
+    'consoles' => [],
+    'filterOptions' => [],
+    'games' => [],
     'noGamesMessage' => 'No games.',
+    'sortOrder' => 'title',
 ])
 
+<?php
+$areFiltersPristine = empty(
+    array_filter($filterOptions, function ($value) {
+        return $value !== false && $value !== 'all';
+    })
+);
+?>
+
 <div>
-    @if (count($consoles) < 1)
-        <p>{{ $noGamesMessage }}</p><br/>
+    @if (count($consoles) < 1 && $areFiltersPristine)
+        <div class="mb-12">
+            <x-empty-state>{{ $noGamesMessage }}</x-empty-state>
+        </div>
```

Comment:
This logic doesn't seem to be working: http://localhost:64000/game/451

I'm assuming it's meant to hide the "no games" message if no filters are applied?

Or maybe this is just a special case where no filters exist.

Reply:
Sorry, I am having trouble understanding. Can you explain this a different way?

For better context, there is a bug currently in prod where if you filter a list down to where there is no content, you can't clear your filter. To repro, visit:

https://retroachievements.org/game/163

Then filter by "Only with achievements". You'll get an empty state, but no way to get back.

Try the same on this branch and you'll have an empty state with the filter options still visible.

Reply:
I just noticed that visiting a Central hub show a big banner saying there's no games.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/41048ff3-aff3-403e-88a6-01b83b444163)

It doesn't have any filter widgets, therefore it's impossible to have filters applied.

I imagine the intent of this code was to not show the banner if there are no games in the unfiltered state, which would be true for these centralized hubs.

Reply:
OK, I think I understand now. I've changed the outer conditional logic to:

```php
@if (!$areFiltersPristine || count($consoles) > 0)
```

With this change, if a hub doesn't have any games (eg: 451), an empty state and meta surface won't display.
However, if a hub does have games and it's filtered to a state where games don't render, the empty state and meta surface both display.